### DISCUSSION TITLE
Add bulk program name capabilities

### DISCUSSION THREAD

**Discussion by conversy** [christofmuc/KnobKraft-orm#215] [General]
The Solaris can be requested a list of program names in a specific bank number, with a program offset. The Solaris will reply with a message that sends 8 names starting from this offset. This allows to quickly (16 requests: 16x8 program names) get the list of programs in a bank, without getting the whole bank.

It might be a peculiar capability, but maybe it's shared by other synths?
A default implementation could just return a list of "friendly program names" to quickly fill up the program list of a bank.

**Comment by christofmuc** [comment]
The E-Mu Morpheus has the same capability. So yes, this is a valid feature idea. The UI currently though never shows the content of a bank without having already downloaded, so we need to find a place to put this.
